compile.sh

```diff
@@ -556,6 +557,9 @@ buildtool="" #Use ninja by default, fall back to make if that is not available.
                -DUSE_XDG=$USE_XDG                              \
                -DUSE_FLTO_IF_AVAILABLE=${USE_FLTO}
 
+    if [ $CORES -gt $MAXCORES ] && [ "$BUILD_TRANSLATIONS" = "ON" ] ; then
+      $RUN $buildtool lang -j $MAXCORES
```

Comment:
I'm not sure it's only the translations. I don't know about codecheck for example in debug builds, and there may be others.

Reply:
Oh, and `$MAXCORES` can be 0 if the detection fails, so you need to check for that case too.